_posts/2023-11-08-securing-wildfly-apps-oidc-openshift.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Securing WildFly Apps with OIDC on OpenShift'
+date: 2023-11-08
+tags: oidc openshift keycloak adapter galleon-pack
+synopsis: Learn how to secure applications deployed to WildFly on OpenShift with OIDC.
+author: theashiot
+link:https://wildfly-security.github.io/wildfly-elytron/blog/securing-wildfly-apps-oidc-openshift/
```

Comment:
You need a space between 'link:' and the URL; otherwise this does not render properly. This isn't an asciidoc link statement; it's front matter for the document processed by Jekyll as key-value pairs.

Reply:
ty, fixed!